docs/shared/node-tutorial/1-code-generation.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Node Tutorial - Part 1: Code Generation'
+title: 'Node Standalone Tutorial - Part 1: Code Generation'
 description: In this tutorial you'll create a backend-focused workspace with Nx.
 ---
```

Comment:
You are using the `frontmatter` here for the first document, but not for the others. Is this intentional?
I think it would be great to have dedicated description and title for the other files too.